k9mail/src/main/java/com/fsck/k9/helper/Contacts.java

```diff
@@ -240,30 +238,13 @@ public Uri getPhotoUri(String address) {
                 if (!c.moveToFirst()) {
                     return null;
                 }
-
-                contactId = c.getLong(CONTACT_ID_INDEX);
+                final String uriString = c.getString(c.getColumnIndex(Photo.PHOTO_URI));
+                return Uri.parse(uriString);
+            } catch (IllegalStateException e) {
```

Comment:
It will happen when the contact have no photo. Maybe there is no need to catch it explicitly, for it will be caught by [the catch below](https://github.com/k9mail/k-9/pull/2315/files/b13dc31555d20a88b3f560058ee07c3a5c781c2e#diff-2bd769df7e1b99efc5dd476c555801fbL267). However, what I concerned is in the original code the [error log](https://github.com/k9mail/k-9/pull/2315/files/b13dc31555d20a88b3f560058ee07c3a5c781c2e#diff-2bd769df7e1b99efc5dd476c555801fbR249) seems only show when something unexpected happens. So I added this catch to avoid the error log. What do you think of it?